ui/app/templates/rule_scheduled_change_modal.html

```diff
@@ -205,10 +205,29 @@ <h4 ng-show= "sc.change_type !== 'delete'">Rule Details</h4>
         </div>
       </div>
       <div class="col-md-6" ng-show= "sc.change_type !== 'delete'">
+        <div class="form-group" ng-class="{'has-error': errors.mig64}">
```

Comment:
Hmm, that's a good point. I'll move it ahead of Alias for consistency - it makes sense to keep all of the fields that affect rule table parsing grouped together.